Gemfile

```diff
@@ -8,5 +8,7 @@ gem 'slather'
 gem 'codestats-metrics-reporter'
 
 # Fastlane and fastlane related scripts gems
-gem 'fastlane'
+gem 'fastlane', '~>2.55.0'
 gem 'rest-client'
+gem 'versionomy'
+gem 'pry'
```

Comment:
I'd remove this gem if we're not actually using it in the code, just to improve bootstrapping times especially in CI.